README.md

```diff
@@ -837,6 +838,141 @@ OPTIONS:
 --timestamps, -t           [Optional] Shows timestamps on each line in the log output.
 ```
 
+## Using Private Registry Authentication
+
+If you want to use privately hosted container images with ECS, the ECS CLI can store your private registry credentials in AWS Secrets Manager and create an IAM role which ECS can use to access the credentials and private images. This allows you to:
+
+* Store private registry credentials within AWS for use with ECS
+* Add the permissions needed to use your registry secrets to a new or existing Task Execution Role
```

Comment:
I think this should read  "Add the permissions needed to add your registry secrets to a new or existing Task Execution Role"